delivery_schenker/views/delivery_schenker_view.xml

```diff
@@ -60,15 +61,21 @@
                             <field name="schenker_own_pickup" />
                             <field name="schenker_pharmaceuticals" />
                         </group>
-                        <group string="Logistics">
-                            <field
-                                name="schenker_default_packaging_id"
-                                attrs="{'required': [('delivery_type', '=', 'schenker')]}"
-                            />
-                            <field
-                                name="schenker_measure_unit"
-                                attrs="{'required': [('delivery_type', '=', 'schenker')]}"
-                            />
+                        <group>
```

Comment:
You don't need this extra group AFAIK. Just add the other group.

Reply:
Without this extra group the new block would be shown on left side, this would be resulting in a big empty block on the right side.

Reply:
Well, I see it as no problem, but OK if you want to leave it that way.